specs/core/1_custody-game.md

```diff
@@ -688,6 +692,18 @@ Append this to `process_final_updates(state)`:
 ```python
     # Clean up exposed RANDAO key reveals
     state.exposed_derived_secrets[current_epoch % EARLY_DERIVED_SECRET_PENALTY_MAX_FUTURE_EPOCHS] = []
+    # Reset withdrawable epochs if challenge records are empty
+    for index, validator in enumerate(state.validator_registry):
+        eligible = True
+        for records in (state.custody_chunk_challenge_records, state.bit_challenge_records):
+            for filter_func in (lambda rec: rec.challenger_index == index, lambda rec: rec.responder_index == index):
+                if len(list(filter(filter_func, records))) > 0:
+                    eligible = False
```

Comment:
early `break`
```suggestion
                    eligible = False
                    break
```

Reply:
I don't think there's much point in the break; it'll only escape the inner of the two loops anyway, and not sure it adds readability.